tensorbay/client/profile.py

```diff
@@ -32,9 +33,8 @@ class Profile:
     """
 
     def __init__(self) -> None:
-        self.summary: DefaultDict[str, DefaultDict[str, Any]] = defaultdict(
-            lambda: defaultdict(int)
-        )
+        manager = Manager()
```

Comment:
the manager process will be enabled even if customer does not need this feature. which is bad, it will be better to put it into `start` method.